lib/vjs-hls.js

```diff
@@ -30,7 +31,49 @@ var attachVideojsStreamrootProvider = function (window, videojs, Hls) {
         }
 
         function _onError(event, data) {
-            console.error('HLS.js error: ' + data.type + ' - fatal: ' + data.fatal + ' - ' + data.details);
+            var error = {
+              message: ('HLS.js error: ' + data.type + ' - fatal: ' + data.fatal + ' - ' + data.details),
+            };
+            console.error(error.message);
+
+            // increment/set error count
+            errorCounts[data.type] ? errorCounts[data.type] += 1 : errorCounts[data.type] = 1;
```

Comment:
You've declared the variable `_errorCounts` but are using `errorCounts` everywhere, so there will be an exception